### PR TITLE
Add get_128bit_scalar function

### PR DESCRIPTION
**Pull request by jonas-lj**
Closing #122

### CODE REVIEW

File: fastcrypto/src/bls12381.rs [hunk 1]
```diff
@@ -254,21 +253,16 @@ impl VerifyingKey for BLS12381PublicKey {
...
-                // Reject zero as it is used for multiplication.
-                vals[0] = rng.next_u64();
-            }
-            let mut rand_i = MaybeUninit::<blst_scalar>::uninit();
-            unsafe {
-                blst_scalar_from_uint64(rand_i.as_mut_ptr(), vals.as_ptr());
-                rands.push(rand_i.assume_init());
-            }
+
+        let mut one = blst_scalar::default();
```
Comment (kchalkias) [inline fastcrypto/src/bls12381.rs]: Let's add a comment that this is an optimization, in accordance to https://github.com/MystenLabs/fastcrypto/issues/120

**Review by kchalkias — APPROVED** [review]
Just a small comment

File: fastcrypto/src/bls12381.rs (hunk 1 above)
Comment (jonas-lj) [reply fastcrypto/src/bls12381.rs]: It was put in this PR by accident, so I've removed it again. Will include it in another PR.

Comment (jonas-lj) [reply fastcrypto/src/bls12381.rs]: @benr-ml just asked me why it is necessary with 128 bits here. The numbers are chosen by the verifier, so statistical security should be enough, right?

**Comment by jonas-lj** [comment]
> Closing #122

@benr-ml just asked me why 128 bits is necessary here. The numbers are chosen by the verifier, so statistical security should be enough?

**Pull request merged**